lib/product.service.ts

```diff
@@ -114,5 +165,93 @@ export class ProductStack extends Stack {
       allowMethods: ['GET', 'OPTIONS'],
       allowHeaders: ['Content-Type']
     });
+    
+    // DynamoDB client
+    const dynamodb = new AWS.DynamoDB.DocumentClient();
```

Comment:
This file is already big. Let's create separate folder with all of this logic for populating the data and:
a) use them here as one method call "populateProductsDb"
Or
b) call specific js method from the npm scripts